<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_GeofeediaAutomation</class>
 <widget class="QMainWindow" name="main_GeofeediaAutomation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geofeedia Automation GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="15,30,15,30">
        <property name="spacing">
         <number>25</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_environment">
          <property name="text">
           <string>Environment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_environment"/>
        </item>
        <item>
         <widget class="QLabel" name="lbl_browser">
          <property name="text">
           <string>Browser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_browser"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="30,70">
        <item>
         <widget class="QTreeView" name="tv_folder_view"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="tb_output"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,95">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_webdriver">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>Configure Webdriver</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
